options/locale/locale_en-US.ini

```diff
@@ -1027,6 +1027,8 @@ file_view_source = View Source
 file_view_rendered = View Rendered
 file_view_raw = View Raw
 file_permalink = Permalink
+file_view_source_blob = View Source Blob
+file_view_rendered_blob = View Rendered Blob
```

Comment:
End users may not know what is `blob`.

Maybe the texts can be shorten to `View Rendered` and `View Source` (or View Origin, I am not sure which is better)